changelog.d/10410.bugfix

```diff
@@ -0,0 +1 @@
+Slight improvements in character set detection for the "URL_Preview" feature.
```

Comment:
```suggestion
Improve character set detection in URL previews by supporting underscores (in addition to hyphens). Contributed by @srividyut.
```

Reply:
Ah, I didn't have enough "work to run trial tests.test_preview.MediaEncodingTestCase".
I will definitely try it.